JavaScript/JavaScriptSDK/appInsights.ts

```diff
@@ -84,6 +84,15 @@ module Microsoft.ApplicationInsights {
             // initialize event timing
             this._eventTracking = new Timing("trackEvent");
             this._eventTracking.action = (name?: string, url?: string, duration?: number, properties?: Object, measurements?: Object) => {
+                if (!measurements) {
+                    measurements = { duration: duration };
```

Comment:
I'm actually not sure about measurements - where will we see it in the portal? (I know that custom dimensions show up below TOP N properties on event details blade). I understand it may be even better than properties if you could do charts based on it (and if dimensions don't support charting). Have you thought about it?